package.json

```diff
@@ -46,7 +46,7 @@
   },
   "dependencies": {
     "@axelar-network/axelar-cgp-solidity": "^4.5.0",
-    "@axelar-network/axelar-gmp-sdk-solidity": "^3.6.1",
+    "@axelar-network/axelar-gmp-sdk-solidity": "^5.0.0",
     "@axelar-network/axelar-local-dev": "^2.0.1",
     "@openzeppelin/contracts": "^4.8.3",
```

Comment:
is openzeppelin dep needed for anything else? if not, we can remove it @npty

Reply:
no, it's safe to delete now